### PR TITLE
Get/set-color excel command

### PR DESCRIPTION
**Pull request by MarekKosinski**

### CODE REVIEW

File: G1ANT.Addon.MSOffice/Commands/ExcelGetColorCommand.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            public IntegerStructure Row { get; set; }
+
+            [Argument(Tooltip = "Cell's column index")]
+            public IntegerStructure ColIndex { get; set; }
+
+            [Argument(Tooltip = "Cell's column name")]
+            public TextStructure ColName { get; set; }
+
+            [Argument(Tooltip = "Name of a variable where the font color's result will be stored")]
+            public VariableStructure FontColorResult { get; set; } = new VariableStructure("fontcolorresult");
```
Comment (pwitusik) [inline G1ANT.Addon.MSOffice/Commands/ExcelGetColorCommand.cs]: rename to FontColor

File: G1ANT.Addon.MSOffice/Commands/ExcelGetColorCommand.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+            [Argument(Tooltip = "Name of a variable where the background color's result will be stored")]
+            public VariableStructure BackgroundResult { get; set; } = new VariableStructure("backgroundcolorresult");
```
Comment (pwitusik) [inline G1ANT.Addon.MSOffice/Commands/ExcelGetColorCommand.cs]: rename to BkColor

File: G1ANT.Addon.MSOffice/Commands/ExcelSetColorCommand.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            public IntegerStructure ColIndex { get; set; }
+
+            [Argument(Tooltip = "Cell's column name")]
+            public TextStructure ColName { get; set; }
+
+            [Argument(Tooltip = "Font color to be set")]
+            public ColorStructure FontColor { get; set; }
+
+            [Argument(Tooltip = "Background color to be set")]
+            public ColorStructure BackgroundColor { get; set; }
```
Comment (pwitusik) [inline G1ANT.Addon.MSOffice/Commands/ExcelSetColorCommand.cs]: rename to BkColor

**Review by pwitusik — CHANGES_REQUESTED** [review]

File: G1ANT.Addon.MSOffice/Commands/ExcelGetColorCommand.cs (hunk 2 above)
Comment (danieladamkaiser) [reply G1ANT.Addon.MSOffice/Commands/ExcelGetColorCommand.cs]: maybe BgColor? :)

File: G1ANT.Addon.MSOffice/Commands/ExcelSetColorCommand.cs [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+        public ExcelSetColorCommand(AbstractScripter scripter) : base(scripter) { }
+
+        public void Execute(Arguments arguments)
+        {
+            object col = null;
+            try
+            {
+                col = ExcelHelper.GetColumn(arguments.ColIndex, arguments.ColName, true);
+                ExcelManager.CurrentExcel.SetColor(arguments.Row.Value, col, arguments.BackgroundColor?.Value, arguments.FontColor.Value);
```
Comment (danieladamkaiser) [inline G1ANT.Addon.MSOffice/Commands/ExcelSetColorCommand.cs]: Missing null conditional operator in: arguments.FontColor.Value

Comment (MarekKosinski) [reply G1ANT.Addon.MSOffice/Commands/ExcelSetColorCommand.cs]: good catch!

**Review by danieladamkaiser — APPROVED** [review]

**Review by pwitusik — APPROVED** [review]

**Pull request merged**